.travis.yml

```diff
@@ -1,10 +1,16 @@
+sudo: false
 language: python
+cache:
+  directories:
+    - $PWD/wheelhouse
 env:
-  - TOXENV=py26
-  - TOXENV=py27
-  - TOXENV=py33
-  - TOXENV=py34
-  - TOXENV=pypy
-  - TOXENV=pypy3
-install: pip install tox
+  - TOXENV=py26 PIP_FIND_LINKS=$PWD/wheelhouse
```

Comment:
I tried doing that .. failed on me when I set a global env and set TOXENV.. I'll try again